### PR TITLE
[kh360-442]View Evaluation Administration Screen-mobile-view

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
[View Evaluation Administration Screen - adjust to make it mobile responsive#442](https://github.com/nerubia/kh360-react/issues/442)

### CODE REVIEW

File: src/components/shared/display-range-date.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <div className='flex gap-3'>
+    <div className='font-bold'>
+      {!isMobile ? <span>{label}</span> : <span>{label.trim().split(" ")[1]}</span>}:
+    </div>
+    {isMobile ? shortenFormatDate(startDate) : convertToFullDate(startDate)} to{" "}
+    {isMobile ? shortenFormatDate(endDate) : convertToFullDate(endDate)}
+  </div>
+)
+
+DateRangeDisplay.propTypes = {
```
Comment (jatnerubia) [inline src/components/shared/display-range-date.tsx:26]: do we need this?

File: src/components/ui/dropdown/dropdown-content.tsx [hunk 2]
```diff
@@ -1,10 +1,22 @@
 interface DropdownContentProps {
   children: React.ReactNode
+  size?: "small" | "medium"
 }
 
-export const DropdownContent = ({ children }: DropdownContentProps) => {
+export const DropdownContent = ({ children, size = "medium" }: DropdownContentProps) => {
+  const getSizeClass = () => {
```
Comment (jatnerubia) [inline src/components/ui/dropdown/dropdown-content.tsx:7]: use `cva`

File: src/utils/format-string.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export function formatString(inputString: string | undefined, maxLength: number): string {
```
Comment (jatnerubia) [inline src/utils/format-string.ts:1]: where do we use this?
I think you can use this [tailwindcss feature](https://tailwindcss.com/docs/text-overflow) 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/components/shared/display-range-date.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/components/shared/display-range-date.tsx:26]: I created this for use across multiple pages, as I have two pages that employ it due to the distinct date formats on desktop and mobile devices.

File: src/components/ui/dropdown/dropdown-content.tsx (hunk 2 above)
Comment (chadiegilnerubia) [reply src/components/ui/dropdown/dropdown-content.tsx:7]: done

File: src/utils/format-string.ts (hunk 3 above)
Comment (chadiegilnerubia) [reply src/utils/format-string.ts:1]: Okey thanks :3 i didn't know there is a class for this hehe

File: src/components/shared/display-range-date.tsx (hunk 1 above)
Comment (jatnerubia) [reply src/components/shared/display-range-date.tsx:26]: I mean, we already have the interface `DateRangeDisplayProps` 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: src/components/shared/display-range-date.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/components/shared/display-range-date.tsx:26]: Ahhhh okey2 getss xD

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**